src/App.js

```diff
@@ -1,10 +1,58 @@
-import Text from "./component/Text";
-import Button from "./component/Button";
-import TextCard from "./component/TextCard";
+import ColorCard from "./component/ColorCard";
+import TextBox from "./component/Text";
+import MediaCard from "./component/MediaCard";
+
 function App() {
   return (
     <>
-      <TextCard text="이게 TextCard...?" button="click" />
+      <TextBox color="blue" fontSize="50px" fontWeight="bold">
+        첫번째 텍스트 박스
+      </TextBox>
+      <TextBox color="red" fontSize="30px" fontWeight="lighter">
+        두번째 텍스트 박스
+      </TextBox>
+
+      <ColorCard color="black" />
+      <ColorCard color="#1E82CD" />
+      <ColorCard color="#FFAAFF" />
+      <ColorCard color="#1CE2B4" />
+      <ColorCard color="#6B1CE2" />
+
+      <MediaCard
+        backgroundImage="https://t4.ftcdn.net/jpg/01/09/88/85/240_F_109888581_d2vqgQnNiNkGRfZPFpIYlS5FIYNBD3ch.jpg"
+        title="Heading"
+        text="This is a short description about the card."
+        backgroundColor="linear-gradient(135deg, #667eea 0%, #764ba2 100%);"
+      ></MediaCard>
```

Comment:
이 코드는 아래와 같이 self-closing 을 활용하여 변경할 수 있을 것 같아요. 한번 찾아볼까요?

```suggestion
      <MediaCard
        backgroundImage="https://t4.ftcdn.net/jpg/01/09/88/85/240_F_109888581_d2vqgQnNiNkGRfZPFpIYlS5FIYNBD3ch.jpg"
        title="Heading"
        text="This is a short description about the card."
        backgroundColor="linear-gradient(135deg, #667eea 0%, #764ba2 100%);"
      />
```